RunningOrder/en.lproj/Localizable.strings

```diff
@@ -85,3 +86,12 @@
 
 "Label" = "Label";
 "Url" = "Url";
+
+// MARK: ToolbarItems
+
+"You can't undo this action." = "You can't undo this action.";
+"Delete the story \"%@\" ?" = "Delete the story \"%@\" ?";
```

Comment:
"=" misalignement

Reply:
there is no alignment convention on this project for now. We might discuss it later :)